tools/drdeploy.c

```diff
@@ -265,8 +265,8 @@ const char *options_list_str =
     "                          wait.  A value of 0 means don't wait for nudges to\n"
     "                          complete."
 #    else  /* WINDOWS */
-    /* FIXME i#840: integrate nudgeunix into drconfig on Unix */
-    "Note: please use the nudgeunix tool to nudge processes on Unix.\n";
+    /* FIXME i#840: integrate drnudgeunix into drconfig on Unix */
+    "Note: please use the drnudgeunix tool to nudge processes on Unix.\n";
```

Comment:
I'm seeing a bunch of other references that need updating, right?
```
$ git grep nudgeunix
api/docs/deployment.dox:nudgeunix tool:
api/docs/deployment.dox:bin32/nudgeunix -pid targetpid -client 0 5
core/lib/dr_events.h: * nudgeunix tool on Linux.  A client in this process can use
core/unix/nudgesig.c: * shared between core & tools/nudgeunix.c
core/unix/nudgesig.c:/* shared with tools/nudgeunix.c */
suite/tests/CMakeLists.txt:        get_target_path_for_execution(toolbindir nudgeunix "${location_suffix}")
suite/tests/runall.cmake:# * nudge = arguments to nudgeunix or drconfig
suite/tests/runall.cmake:  set(nudge_cmd nudgeunix)
suite/tests/runall.cmake:  # nudgeunix and drconfig have different syntax:
tools/CMakeLists.txt:    add_executable(nudgeunix nudgeunix.c ${PROJECT_SOURCE_DIR}/core/unix/nudgesig.c)
tools/CMakeLists.txt:    DR_install(TARGETS nudgeunix DESTINATION ${INSTALL_BIN})
tools/drdeploy.c:    /* FIXME i#840: integrate nudgeunix into drconfig on Unix */
tools/drdeploy.c:    "Note: please use the nudgeunix tool to nudge processes on Unix.\n";
tools/nudgeunix.c: * nudgeunix.c: nudges a target Linux process
tools/nudgeunix.c:    "usage: nudgeunix [-help] [-v] [-pid <pid>] [-type <type>] [-client <ID> <arg>]\n"
tools/nudgeunix.c:            printf("nudgeunix version %s -- build %d\n", STRINGIFY(VERSION_NUMBER),
```

Reply:
This grep is run on a different source tree. But you are right. I still missed a couple of references.